odbc/src/api/col_attr_describe_tests.rs

```diff
@@ -422,19 +513,183 @@ mod unit {
                         buffer_length,
                         out_length,
                         numeric_attr_ptr,
-                    )
+                    ),
+                    "expected success but got failure"
+                );
+                assert_eq!(
+                    expected, *numeric_attr_ptr,
+                    "expected {} but got {}",
+                    expected, *numeric_attr_ptr
                 );
-                assert_eq!(expected, *numeric_attr_ptr);
                 let _ = Box::from_raw(char_buffer as *mut WChar);
             }
         }
+        unsafe {
+            let _ = Box::from_raw(conn as *mut WChar);
+            let _ = Box::from_raw(env as *mut WChar);
+        }
+    }
+
+    #[test]
+    fn test_numeric_field_attributes_odbc_2() {
```

Comment:
similar q -- if odbc 2 only effects datetime types + this has the same expectations, we might be able to remove this test case

Reply:
ditto, will remove